pkg/util/util.go

```diff
@@ -147,3 +152,66 @@ func EnsureValidName(name string) string {
 		return c
 	}, name)
 }
+
+func DeleteXfrmRules() error {
```

Comment:
I suggest we put this in the `netlink` package which is more meaningful than `util`.

Reply:
Done